.claude/CLAUDE.md

```
@@ -0,0 +1,259 @@
+- **Framework**: React 또는 Next.js (TBD)
+- **Language**: TypeScript
+- **Styling**: Tailwind CSS (TBD)
+- **UI**: Control UI vs Treatment UI
+
+## Code Organization
+
+### Backend Architecture
+```
+backend/src/main/java/foodiepass/server/
+├── menu/              # ✅ 재사용 (MenuService, OCR, Enrichment)
+├── language/          # ✅ 재사용 (Translation)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)